app/src/main/java/com/example/cleanarchitecture/viewmodel/CharacterInformationViewModel.kt

```diff
@@ -0,0 +1,35 @@
+package com.example.cleanarchitecture.viewmodel
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.example.cleanarchitecture.utils.Event
+import com.example.domain.entity.CharacterData
+
+class CharacterInformationViewModel : ViewModel() {
+
+    private var mutableComicsState: MutableLiveData<Event<FragmentData>> = MutableLiveData()
+    val liveDataState: LiveData<Event<FragmentData>>
+        get() = mutableComicsState
```

Comment:
you're creating a useless livedata (liveDataState) please convert this into a function

Reply:
done